authui/src/authflowv2/base.css

```diff
@@ -36,15 +36,21 @@
     --color-error: #ff295b;
     --color-warning: #f39d17;
 
-    /* primary */
-    --color-primary-base: #176df3;
-    --color-primary-100: #e2e7fe;
-    --color-primary-200: #adbdfd;
-    --color-primary-300: #7295fc;
-    --color-primary-400: #176df3;
-    --color-primary-500: #0d4cae;
-    --color-primary-600: #052e6f;
-    --color-primary-700: #011234;
+    /* Primary colors is generated using fluent ui theme generator */
+    /* https://developer.microsoft.com/en-us/fluentui?fabricVer=5#/styles/themegenerator */
+    /* Parameters:
+      Primary color: #176df3
+    */
+
+    --color-primary-theme-darker: #0c3c88;
+    --color-primary-theme-dark: #1151b8;
```

Comment:
I tested a bit with the theme generator. When the input color is light enough, then `light` is NOT actually light, but visually darker than the input color. So the name `light`, `lighter`, `lighter-alt` may be confusing.

If we use `-primary` and `-dark` only, maybe we should only include these two in our palatte?

Reply:
The name of the color token is defined by fluent ui palatte.
I think I can remove the other colors first.